imap_processing/codice/l0/__init__.py

```diff
@@ -0,0 +1,515 @@
+LOSSY_A_TABLE = {
```

Comment:
Are these tables all sequential lists? You could just make them a list and still index into them in the same way without needing the keys() stored too. `LOSS_A_TABLE = [1, 2, 3, 4, ...]`

Reply:
:+1: Sounds good, just something to keep in mind in the future. My guess is you might be wanting to turn these into numpy arrays down the road and then you could send multiple compressed values at once into these to do faster lookups all at the same time rather than a Python for-loop for each individual value.